tests/test_lsst.py

```diff
@@ -374,7 +374,7 @@ def test_pupil_coords_from_pixel_coords(self):
         chip_name_possibilities = ('R:0,1 S:1,1', 'R:0,3 S:0,2', 'R:4,2 S:2,2', 'R:3,4 S:0,2')
 
         chip_name_list = [chip_name_possibilities[ii]
-                          for ii in rng.random_integers(0,3,n_samples)]
+                          for ii in rng.randint(0,3,n_samples)]
```

Comment:
I'm pretty sure this is wrong.  Or at least it is a change in what the test is doing to something now what you meant to do.

[numpy.random_integers](http://docs.scipy.org/doc/numpy/reference/generated/numpy.random.random_integers.html) selects from a closed range, [0,3] here, while [numpy.randint](http://docs.scipy.org/doc/numpy/reference/generated/numpy.random.randint.html) selects from a half-open range, [0,3) here.

Since the former is now deprecated (why, numpy?), I think you want to change the argument to `(0,4,n_samples)` for `randint`.